acme/acme/crypto_util.py

```diff
@@ -373,24 +376,50 @@ def get_names_from_subject_and_extensions(
 
 def _pyopenssl_cert_or_req_all_names(loaded_cert_or_req: Union[crypto.X509, crypto.X509Req]
                                      ) -> List[str]:
+    """
+    Deprecated
+    .. deprecated: 3.x.x
```

Comment:
let's go with 4.0 for now

```suggestion
    .. deprecated: 4.0.0
```